src/EditorFeatures/CSharpTest/GenerateFromMembers/AddConstructorParametersFromMembers/AddConstructorParametersFromMembersTests.cs

```diff
@@ -297,7 +298,7 @@ public Program((int, string) i)
     (int, string) i;
     (string, int) s;
 
-    public Program((int, string) i, (string, int) s = default)
+    public Program((int, string) i, (string, int) s = default((string, int)))
```

Comment:
can you please update these tests to explicitly use c# 7.1, where default literals were introduced?

Reply:
I [did](https://github.com/dotnet/roslyn/pull/27054) that for some of the tests. But I don't think it matters or is harmful to leave some tests that are sensitive to `default` language version.